src/payment/strategies/googlepay/googlepay-payment-processor.ts

```diff
@@ -232,4 +233,12 @@ export default class GooglePayPaymentProcessor {
             this._billingAddressActionCreator.updateAddress(googlePayAddressMapped)
         );
     }
+
+    private _getFirstAndLastName(name: string[]): string[] {
```

Comment:
When defining a tuple return type it's best to explicitly declare it like:

`[string, string]`

That way people see an error if they try to deconstruct elements that don't exist:

`const [firstName, lastName, somethingElse] = this._getFirstAndLastName(name);`